server/chat-plugins/uno.ts

```diff
@@ -216,29 +206,21 @@ class UnoGame extends Rooms.RoomGame {
 
 	/**
 	 * Overwrite the default makePlayer so it makes an UnoGamePlayer instead.
-	 * @param {User} user
-	 * @return {UnoGamePlayer}
 	 */
-	makePlayer(user) {
+	makePlayer(user: User): UnoGamePlayer {
```

Comment:
Generally we don't annotate return types unless typescript can't figure it out itself.